packages/invoices/src/types.ts

```diff
@@ -86,3 +134,45 @@ export type OutboundShipmentSummaryItem = {
   note?: string | null;
   isDeleted?: boolean;
 };
+
+export interface OutboundShipment extends Invoice {
+  items: OutboundShipmentSummaryItem[];
+  status: OutboundShipmentStatus;
+  update?: <K extends keyof Invoice>(key: K, value: Invoice[K]) => void;
+  upsertLine?: (line: OutboundShipmentRow) => void;
+  deleteLine?: (line: OutboundShipmentRow) => void;
+}
+export interface InboundShipmentRow extends InvoiceLine {
+  updateNumberOfPacks?: (quantity: number) => void;
+  stockLineId: string;
+  invoiceId: string;
+  itemId: string;
+  isUpdated?: boolean;
+  isDeleted?: boolean;
+  isCreated?: boolean;
+}
+
+export type InboundShipmentItem = {
+  id: string;
+  itemId: string;
+  itemCode: string;
+  itemName: string;
+  unitQuantity: number;
+  numberOfPacks: number;
+  locationDescription?: string | null;
+  itemUnit?: string;
+  batch?: string | null;
+  batches: Record<string, OutboundShipmentRow>;
+  sellPrice?: number | undefined;
```

Comment:
I was wondering what the differences are between In & Out BoundShipments, and here's one! Outbound will show `sellPrice` and Inbound should show `costPrice`.

All the same, both types can have both props - it's only the column list which needs to differ!

Reply:
Yeah there's really not much differences right now and maybe not ever, maybe it's backwards but if it turns out there's no difference we can consolidate